app/grants/models/grant.py

```diff
@@ -18,12 +18,69 @@
 from townsquare.models import Favorite
 from web3 import Web3
 
+from .clr_match import CLRMatch
 from .contribution import Contribution
 from .grant_clr_calculation import GrantCLRCalculation
 from .grant_collection import GrantCollection
 from .subscription import Subscription
 
 
+class GrantPayout(SuperModel):
+    PAYOUT_STATUS = [
+        ('pending', 'pending'),
+        ('ready', 'ready'),
+        ('expired', 'expired'),
+        ('funding_withdrawn', 'funding_withdrawn')
+    ]
+    NETWORKS = [
+        ('mainnet', 'mainnet'),
+        ('rinkeby', 'rinkeby'),
+        # ('polygon', 'polygon'),
+        # ('mumbai', 'mumbai'),
+    ]
+
+    name = models.CharField(
+        max_length=25,
+        help_text=_('Display Name for Payout')
+    )
+    contract_address = models.CharField(
+        max_length=255,
+        null=True,
+        blank=True,
+        help_text=_('Payout Contract from which funds would be claimed')
+    )
+    network = models.CharField(
+        max_length=15,
+        default='mainnet',
+        choices=NETWORKS,
+        help_text=_('Network where contract is deployed')
+    )
+    payout_token = models.CharField(
+        max_length=10,
+        default='DAI',
+        help_text=_('Currency in which funds would be paid')
+    )
+    # payout_token_address = models.CharField(
```

Comment:
this is commented out for now due as we support only DAI payouts but that will change post this round